tests/stub/retry/test_retry_clustering.py

```diff
@@ -182,6 +182,40 @@ def twice(tx):
         self._routingServer.done()
         self._readServer.done()
 
+    def test_close_connection_on_retriable_cluster_error(self):
+        num_retries = 0
+
+        def work(tx):
+            nonlocal num_retries
+            num_retries += 1
+            result = tx.run("RETURN " + str(num_retries))
+            record = result.next()
+            return record.values[0]
+
+        self._routingServer.start(
+            path=self.script_path("clustering", "router_retry_once.script"),
+            vars=self.get_vars()
+        )
+        vars = {
+            "#ERROR#": "Neo.ClientError.Cluster.NotALeader",
+        }
+        self._writeServer.start(
+            path=self.script_path("retry_with_fail_after_run_server.script"),
+            vars=vars
+        )
```

Comment:
```suggestion
        self._writeServer.start(
            path=self.script_path("retry_with_fail_after_run_server.script"),
            vars={"#ERROR#": "Neo.ClientError.Cluster.NotALeader"}
        )
```
just a tad more compact